Utility/Utility.Test/ExceptionTest.cpp

```diff
@@ -4,21 +4,60 @@
 
 using Catch::Contains;
 
-[[noreturn]] static void ExceptionThrowerMock(){
+[[noreturn]] static void ExceptionThrowerMock()
+{
     throw Utility::NotImplementedException("Not implemented", __FILE__, __LINE__);
 }
 
-[[noreturn]] static void OperationDBExceptionMock() { 
+void NoExceptionMock()
+{
+    Utility::NotImplementedException("Implemented", __FILE__, __LINE__);
+}
+
+[[noreturn]] static void OperationDBExceptionMock()
+{
     throw Utility::OperationDBException("DB error operation", __FILE__, __LINE__); 
 }
 
+void OperationDBNoExceptionMock()
+{
+    Utility::OperationDBException("DB success operation", __FILE__, __LINE__);
+}
+
 TEST_CASE("Test NotImplementedException format", "[Utility.Exception]")
 {
-    REQUIRE_THROWS_WITH(ExceptionThrowerMock(), Contains("Not implemented") && Contains("ExceptionTest.cpp"));
+    SECTION("Exception is expected")
+    {
+        REQUIRE_THROWS_WITH(ExceptionThrowerMock(), Contains("Not implemented") && Contains("ExceptionTest.cpp"));
+    }
 
+    SECTION("Exception is not expected")
+    {
+        REQUIRE_NOTHROW(NoExceptionMock());
+    }
 }
 
 TEST_CASE("Test OperationDBException format", "[Utility.Exception]")
 {
-    REQUIRE_THROWS_WITH(OperationDBExceptionMock(), Contains("DB error operation") && Contains("ExceptionTest.cpp"));
+    SECTION("Exception is expected")
+    {
+        REQUIRE_THROWS_WITH(OperationDBExceptionMock(), Contains("DB error operation") && Contains("ExceptionTest.cpp"));
+    }
+
+    SECTION("Exception is not expected")
+    {
+        REQUIRE_NOTHROW(OperationDBNoExceptionMock());
+    }
+}
+
+TEST_CASE("Test formatExceptionMessage")
+{
+    std::string message  = "Test exception";
+    std::string filename = "ExpectionFile.txt";
```

Comment:
Why not string_view? You've changed a string on string_view in another file but there you're still using std::string

Reply:
What 'another' file do you mean? And there is where?

Reply:
It's in the Utility.hpp, but I've seen that regex doesn't have a overriding with string_view